src/index.js

```diff
@@ -7,37 +17,160 @@ import Sender from './sender'
- * @param {String} params.source_password
- * @param {String} params.destination_ledger - Ledger URI
- * @param {String} params.destination_account - Account URI
- * @param {String} params.destination_username
- * @param {String} params.destination_amount - Amount (a string, so as not to lose precision)
+ * @param {URI} params.sourceAccount - Account URI
+ * @param {String} params.sourceUsername
+ * @param {String} params.sourcePassword
+ * @param {URI} params.destinationAccount - Account URI
+ * @param {String} params.destinationAmount - Amount (a string, so as not to lose precision)
+ *
+ * Required for Atomic mode only:
+ * @param {URI} params.notary - Notary URI (if provided, use Atomic mode)
+ * @param {String} params.notaryPublicKey - Base64-encoded public key
+ *
+ * Other:
+ * @param {String} params.destinationMemo
+ * @param {Condition} params.receiptCondition - Object, execution condition.
+ *                                              If not provided, one will be generated.
+ */
+export default async function sendPayment (params) {
+  const subpayments = await findPath({
+    sourceAccount: params.sourceAccount,
+    destinationAccount: params.destinationAccount,
+    destinationAmount: params.destinationAmount
+  })
+  await executePayment(subpayments, {
+    sourceAccount: params.sourceAccount,
+    sourceUsername: params.sourceUsername,
+    sourcePassword: params.sourcePassword,
+    notary: params.notary,
+    notaryPublicKey: params.notaryPublicKey,
+    destinationMemo: params.destinationMemo,
+    receiptCondition: params.receiptCondition
+  })
+}
+
+/**
+ * Execute a transaction.
+ *
+ * @param {[Object]} _subpayments - The quoted payment path.
+ * @param {Object} params
+ *
+ * Required for both modes:
+ * @param {URI} params.sourceAccount - Account URI
+ * @param {String} params.sourceUsername
+ * @param {String} params.sourcePassword
  *
  * Required for Atomic mode only:
- * @param {String} params.notary - Notary URI (if provided, use Atomic mode)
- * @param {String} params.notary_public_key - Base64-encoded public key
- * @param {String} params.receipt_condition - Object, execution condition.
- *                                            If not provided, one will be generated.
+ * @param {URI} params.notary - Notary URI (if provided, use Atomic mode)
+ * @param {String} params.notaryPublicKey - Base64-encoded public key
  *
  * Other:
- * @param {[Object]} params.subpayments - The quoted payment path. If provided, the
- *                                        pathfinding/quoting steps will be skipped.
+ * @param {String} params.destinationMemo
+ * @param {Condition} params.receiptCondition - Object, execution condition.
+ *                                              If not provided, one will be generated.
  */
-export default async function (params) {
-  const sender = new Sender(params)
-  if (!params.subpayments) {
-    await sender.findPath()
+export async function executePayment (_subpayments, params) {
```

Comment:
What version of the connector are you running? I think you may have an old one, the current format returned includes `source_transfers.debits` with an `"account"` of `null`. Also, be sure to pass `destination_account` in to the `/quote` so that the `destination_transfers.credits` is filled in as well.